app/partials/showModule.html

```diff
@@ -76,6 +76,17 @@ <h2 ng-if="editable">
       </tr>
 
       <tr>
+
+        <th>Required Modulus</th>
```

Comment:
Should be "Required Modules" ;)
Modulus = the platform that serves modules
(I make this same typo all the time though)